### ISSUE TITLE
[REQUEST] Theme Engine

### ISSUE THREAD

**Issue by hahagu** [serghey-rodin/vesta#1177]
Why doesn't VestaCP have a theme engine even though many people want it?
Other panels, like Sentora has a theme engine which allows people to customize their panel's looks.
I prefer VestaCP in every way over all other panels, except the fact that it doesn't have a theme engine.
I'd really like to see a theme engine on VestaCP on the next update, as well as an active development. (Perhaps some API and documentation may help?)
Thanks!

**Comment by Skamasle** [comment]
Will be nice yes, but as vesta is opensource, you can modify all you need :)

Open a new idea in bug tracker 👍

**Comment by anton-reutov** [comment]
Yes. We have a plan to make this feature, but I can't say when it will be implemented

**Comment by hahagu** [comment]
Thanks! Although it's okay not to have it later, but I hope to see this in the next update!

**Comment by hahagu** [comment]
BugTracker LINK:
https://bugs.vestacp.com/issues/536
Please vote for this! (Click on green arrow)
